data/costs.csv

```diff
@@ -181,7 +183,7 @@ decentral solar thermal,2030,lifetime,20,years,HP;
 central solar thermal,2030,FOM,1.4,%/year,HP;
 central solar thermal,2030,investment,140000,EUR/1000m2,HP;
 central solar thermal,2030,lifetime,20,years,HP;
-HVAC overhead,2030,investment,400,EUR/MW/km,Hagspiel;
+HVAC overhead,2030,investment,40,EUR/MW/km,Hagspiel;
```

Comment:
Mmmm it would be better to add a line on MVAC for example and use it, rather than editing HVAC

Reply:
Not sure to have understand what to do there, sorry!

Reply:
Restore the old data for HVAC and add a line for MVAC